src/converters/xtbConverter.ts

```
@@ -171,9 +177,27 @@ export class XtbConverter extends AbstractConverter {
 
                 const match = record.comment.match(/(?:OPEN|CLOSE) BUY ([0-9]+(?:\.[0-9]+)?(?:\/[0-9]+(?:\.[0-9]+)?)?) @ ([0-9]+(?:\.[0-9]+)?)|(?:[A-Z\. ]+) ([0-9]+(?:\.[0-9]+)?)/)
 
-                let quantity = parseFloat(match[1]?.split("/")[0]);
-                let unitPrice = parseFloat(match[2]);
-                const dividendPerShare = parseFloat(match[3]);
+                let quantity = 0;
+                let unitPrice = 0;
+                let dividendPerShare = 0;
+
+                if (match && match?.[1] && match?.[2]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)